lib/modules/settings/screens/notification.dart

```diff
@@ -47,41 +49,41 @@ class _NotificationScreenState extends State<NotificationScreen> {
 
   @override
   Widget build(BuildContext context) {
-    // loading
     if (permission == null) {
       return const Center(
         child: CircularProgressIndicator(),
       );
     }
 
-    var t = AppLocalizations.of(context);
+    var t = AppLocalizations.of(context)!;
 
     SettingsSection settingsSection;
     if (Platform.isIOS && !permission!) {
-      settingsSection = SettingsSection(title: Text(t!.prayersAlerts), tiles: [
+      settingsSection = SettingsSection(title: Text(t.prayersAlerts), tiles: [
         SettingsTile.switchTile(
           initialValue: permission,
           onToggle: (value) {
-            // open settings
             openAppSettings().then((_) {
               Navigator.pop(context);
             });
           },
           title: Text(t.allowNotifications),
-          leading: const Icon(
-            Icons.notifications_off,
-          ),
+          leading: const Icon(Icons.notifications_off),
         )
       ]);
     } else {
       settingsSection = SettingsSection(
-        title: Text(t!.prayersAlerts),
+        title: Text(t.prayersAlerts),
         tiles: [
-          _buildPrayerSettingsTile(PrayerType.fajr, t.fajr),
-          _buildPrayerSettingsTile(PrayerType.duhur, t.duhur),
-          _buildPrayerSettingsTile(PrayerType.asr, t.asr),
-          _buildPrayerSettingsTile(PrayerType.maghrib, t.maghrib),
-          _buildPrayerSettingsTile(PrayerType.isha, t.isha),
+          _buildSettingsTile(
```

Comment:
maybe comment this since its not finished

Reply:
Actually I was planning to start working on it right after this.

Reply:
Ah ok, I thought it was ready for review

Reply:
I mean in a different pr